MAVProxy/modules/mavproxy_paramedit/param_editor.py

```diff
@@ -155,6 +156,28 @@ def process_mavlink_packet(self, m):
                 del self.paramchanged[m.param_id.upper()]
             self.gui_event_queue.put(ParamEditorEvent(
                 ph_event.PEGE_WRITE_SUCC, paramid=m.param_id.upper(), paramvalue=m.param_value))
+        if mtype == 'RC_CHANNELS_RAW':
```

Comment:
why not use:
  if mtype in ['RC_CHANNELS_RAW', 'RC_CHANNELS']
as its just copy&paste

Reply:
It's because of channel numbers. Both contain different number of channels. I will change it to remove code duplication to as much extent as possible.

Reply:
Done.